app/assets/javascripts/govuk_publishing_components/analytics-ga4/pii-remover.js

```diff
@@ -2,7 +2,7 @@
   'use strict'
 
   var GOVUK = global.GOVUK || {}
-  var EMAIL_PATTERN = /[^\s=/?&#]+(?:@|%40)[^\s=/?&]+/g
+  var EMAIL_PATTERN = /[^\s=/?&#+]+(?:@|%40)[^\s=/?&+]+/g
```

Comment:
Not sure I fully understand the implications of this change, might be worth discussing directly.

Also should there be a test for this change?

Reply:
Thanks, that was helpful. I think this change is good, because it's better to prevent the regex from escaping from the email address as you describe.

The gmail address problem is a concern, but at least we're redacting the majority part of the email, which I think should make it anonymous enough. For example `john+work@gmail.com` (stemming from the original `john@gmail.com`) would be redacted to `john+[email]`.

So I think we're okay, add a test and we should be good 👍